### ISSUE TITLE
API / JSON output

### ISSUE THREAD

**Issue by djoey123** [yanikore/phpBrute#2]
If you have time maybe could you make an single proxy checker as.

php phpBrute.php -m ProxyChecker -i proxy:port

as output:
1 = working
0 = not working

or some tips how i can make this,

tnx 👍

**Comment by yani** [comment]
Yeah I've been wanting to add a flag like -api or -json which would return the output in said format. But that's for the future.

Right now you can directly include the ModuleFactory php class. It might be hard if you have not much PHP experience, but I made it in a way that would allow you to use a module even from normal website.
